plugin.video.drnu/addon.xml

```diff
@@ -11,16 +11,16 @@
     <extension point="xbmc.addon.metadata">
         <summary lang="en">Watch live and archived TV from DR (Danish Broadcasting Corporation)</summary>
         <summary lang="da">Se live og arkiveret TV fra DR</summary>
-        <description lang="en">All DR's live TV channels and archived programs as seen on dr.dk/tv.[CR][CR]If you have comments or suggestions for this addon, please feel free to participate in the debate on my blog at http://tommy.winther.nu</description>
-        <description lang="da">Alle DR's live TV kanaler samt alle de programmer du finder på dr.dk/tv.[CR][CR]Har du kommentarer, ris eller ros til denne addon er du velkommen til at deltage i debatten på min blog på http://tommy.winther.nu</description>
+        <description lang="en">All DR's live TV channels and archived programs as seen on dr.dk/tv.[CR][CR]If you have comments or suggestions for this addon, please feel free to participate in the KODI official forums</description>
+        <description lang="da">Alle DR's live TV kanaler samt alle de programmer du finder på dr.dk/tv.[CR][CR]Har du kommentarer, ris eller ros til denne addon er du velkommen til at deltage i debatten på KODI forummet</description>
         <disclaimer lang="en">Some parts of this addon may not be legal in your country of residence - please check with your local laws.</disclaimer>
         <disclaimer lang="da">Nogle dele af denne addon er muligvis ikke lovlig i dit land - kontroller venligst dine lokale love.</disclaimer>
         <license>GPL 2.0</license>
         <platform>all</platform>
         <language>da</language>
-        <email>tommy@winther.nu</email>
-        <website>http://tommy.winther.nu</website>
-        <forum>http://tommy.winther.nu/wordpress/forums/forum/addons/dr-nu-player/</forum>
+        <email>msj33@hotmail.com</email>
+        <website>https://github.com/xbmc-danish-addons/plugin.video.drnu</website>
```

Comment:
Website should list the site where the content is comming from.
So in this case something like dr.dk.

Reply:
Fixed

Reply:
No, you now mixed source and website.
source is source code repo and was correct before
website is content website

Reply:
Your completely right :D - Fixed